libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -112,17 +202,65 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
     }
 
     /** @hidden */
-    private _setContentTemplate(): void {
+    private _setContentTemplates(): void {
+        const templatesLength = this.contentTemplates.length;
+        this.contentTemplates = [];
+        let _stepId = 0;
         this.steps.forEach((step) => {
+            if (step.content) {
+                step.content.tallContent = false;
+                step.content.wizardContentId = _stepId.toString();
```

Comment:
Do you need `wizardContentId`? I see it's used only to add id for content element, but this id will be just a number, so it will for sure be redundant.

Reply:
It is necessary to keep the correct step indicator status when the user is scrolling through steps